src/Test.FunctionalTests.Sarif/TestData/Partitioning/Partition/ExpectedOutputs/TrivialPartitionFunction.sarif

```diff
@@ -51,6 +51,9 @@
       "results": [
         {
           "ruleId": "TST0001",
+          "message": {
+            "text": "Message 1.1."
+          },
```

Comment:
thanks for taking care of these. when i rebaselined locally, a bunch of these files were popping up changed. :)